src/opentracing/tracer.js

```diff
@@ -121,6 +123,10 @@ class SignalFxTracer extends Tracer {
     }
     return flushed
   }
+
+  getRecordedValueMaxLength () {
+    return this._recordedValueMaxLength
+  }
```

Comment:
Don't see the need for this since this is an internal feature and not to be exposed as part of span API. We can directly access `_recordedValueMaxLength`